tests/meltano/core/test_venv_service.py

```diff
@@ -117,6 +117,18 @@ async def test_requires_clean_install(self, project, subject: VenvService):
         # Make sure the venv exists already
         await subject.install(["example"], clean=True)
 
+        original_link_target = subject.python_path.readlink()
```

Comment:
This was added in 3.9, so maybe we can port [the current 3.11 implementation](https://github.com/python/cpython/blob/3.11/Lib/pathlib.py#L1081-L1087):

```python
import os
import sys
from pathlib import Path

if sys.version_info < (3, 11):
    readlink = Path.readlink()
else:
    def readlink(self):
        """Return the path to which the symbolic link points."""
        if not hasattr(os, "readlink"):
            raise NotImplementedError("os.readlink() not available on this system")
        return self._from_parts((os.readlink(self),))

...

        original_link_target = readlink(python_path)
```